### PR TITLE
Babylonian_Method in Kotlin

### LINKED ISSUE #1168: Babylonian_method in kotlin
Please assign it to me. @jainaman224

### PR DESCRIPTION
**Pull request by shubhammantri1**
Fixes #1168   

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Adding Babylonian_Method in Kotlin

#### Languages Used:
- Kotlin

#### Files Added:
- Babylonian_Method.kt

### CODE REVIEW

File: Babylonian_Method/Babylonian_Method.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+fun square_root(num: Float):Float {
+    var number = num
+    var guess = 1F
+    var accuracy = 0.000001
+    while(number - guess > accuracy) {
```
Comment (MastersAbh) [inline Babylonian_Method/Babylonian_Method.kt]: space after while

File: Babylonian_Method/Babylonian_Method.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+//Driver Function for Babylonian
+fun main() {
+    //let number be 90
+    var num = 90
+    println("Square Root of $num is ${square_root(num.toFloat())}")
+}
+
+//Output: Square Root of 90 is 9.486834
```
Comment (MastersAbh) [inline Babylonian_Method/Babylonian_Method.kt]: space after //

File: Babylonian_Method/Babylonian_Method.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    while(number - guess > accuracy) {
+        number = (number + guess) / 2
+        guess = num / number
+    }
+    return number
+}
+
+//Driver Function for Babylonian
+fun main() {
+    //let number be 90
```
Comment (MastersAbh) [inline Babylonian_Method/Babylonian_Method.kt]: space after //

File: Babylonian_Method/Babylonian_Method.kt [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    var guess = 1F
+    var accuracy = 0.000001
+    while(number - guess > accuracy) {
+        number = (number + guess) / 2
+        guess = num / number
+    }
+    return number
+}
+
+//Driver Function for Babylonian
```
Comment (MastersAbh) [inline Babylonian_Method/Babylonian_Method.kt]: space after //

File: Babylonian_Method/Babylonian_Method.kt (hunk 1 above)
Comment (shubhammantri1) [reply Babylonian_Method/Babylonian_Method.kt]: Done

File: Babylonian_Method/Babylonian_Method.kt (hunk 2 above)
Comment (shubhammantri1) [reply Babylonian_Method/Babylonian_Method.kt]: Done

File: Babylonian_Method/Babylonian_Method.kt (hunk 3 above)
Comment (shubhammantri1) [reply Babylonian_Method/Babylonian_Method.kt]: Done

File: Babylonian_Method/Babylonian_Method.kt (hunk 4 above)
Comment (shubhammantri1) [reply Babylonian_Method/Babylonian_Method.kt]: Done

**Pull request merged**